site/docs/use-cases/image-processing/_sections/_run_model/index.mdx

```diff
@@ -29,6 +29,16 @@ It can generate text from a text prompt and images as inputs.
     </TabItemCpp>
 </LanguageTabs>
 
+The prompt can contain `<ov_genai_image_i>` with `i` replaced with an actual zero based index to refer to an image. Reference to images used in previous prompts isn't implemented. A model's native image tag can be used instead of `<ov_genai_image_i>`. These tags are:
+1. InternVL2: `<image>\n`
+2. llava-1.5-7b-hf: `<image>`
+3. LLaVA-NeXT: `<image>`
+4. MiniCPM-V-2_6: `(<image>./</image>)\n`
+5. Phi-3-vision: `<|image_i|>\n` - the index starts with one
+6. Qwen2-VL: `<|vision_start|><|image_pad|><|vision_end|>`
+
+If the prompt doesn't contain image tags, but images are provided, the tags are prepended to the prompt.
+
```

Comment:
I would suggest moving this to separate subsection (e.g. "Using image tags in prompt") under "Additional Usage Options" section

Reply:
Moved https://wovchena.github.io/openvino.genai-public/docs/use-cases/image-processing/